tools/releases/build-info-flags.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+versionInfo=$("${BASH_SOURCE%/*}"/version.sh)
+
+IFS=' ' read -r -a values <<< "$versionInfo"
+
+fields=( "version" "gitTag" "gitCommit" "buildDate" "Envoy" )
+
+for index in "${!fields[@]}"
+do
+   echo -n "-X github.com/kumahq/kuma/pkg/version.${fields[index]}=${values[index]} "
+done
```

Comment:
Using this inline in the makefile would also avoid the double call to `version.sh`

Reply:
👍